libs/utils/analysis/tasks_analysis.py

```diff
@@ -200,6 +200,10 @@ def plotTasks(self, tasks=None, signals=None):
         Tasks PELT signals:
                 load_sum, util_sum, period_contrib, sched_overutilized
 
+        Addidional custom signals can be specified and they will be represented
+        in the "Task signals plots" if they represent valid keys of the task
+        load/utilization trace event (e.g. sched_load_avg_task).
+
```

Comment:
I tried this with `signals=['util_avg_pelt'] and got this error:

```
AssertionErrorTraceback (most recent call last)
<ipython-input-31-a271acb5b7ec> in <module>()
----> 1 trace.analysis.tasks.plotTasks(tasks=['many0', 'many1'], signals=['util_avg_pelt'])

/home/brendan/sources/lisa/libs/utils/analysis/tasks_analysis.py in plotTasks(self, tasks, signals)
    336                       .format(self._trace.plots_dir, self._trace.plots_prefix,
    337                               tid, task_name)
--> 338             pl.savefig(figname, bbox_inches='tight')
    339 
    340     def plotBigTasks(self, max_tasks=10, min_samples=100,

/usr/lib/pymodules/python2.7/matplotlib/pyplot.pyc in savefig(*args, **kwargs)
    559 def savefig(*args, **kwargs):
    560     fig = gcf()
--> 561     return fig.savefig(*args, **kwargs)
    562 
    563 

/usr/lib/pymodules/python2.7/matplotlib/figure.pyc in savefig(self, *args, **kwargs)
   1419             self.set_frameon(frameon)
   1420 
-> 1421         self.canvas.print_figure(*args, **kwargs)
   1422 
   1423         if frameon:

/usr/lib/pymodules/python2.7/matplotlib/backend_bases.pyc in print_figure(self, filename, dpi, facecolor, edgecolor, orientation, format, **kwargs)
   2167                     **kwargs)
   2168                 renderer = self.figure._cachedRenderer
-> 2169                 bbox_inches = self.figure.get_tightbbox(renderer)
   2170 
   2171                 bbox_artists = kwargs.pop("bbox_extra_artists", None)

/usr/lib/pymodules/python2.7/matplotlib/figure.pyc in get_tightbbox(self, renderer)
   1562                 bb.append(ax.get_tightbbox(renderer))
   1563 
-> 1564         _bbox = Bbox.union([b for b in bb if b.width != 0 or b.height != 0])
   1565 
   1566         bbox_inches = TransformedBbox(_bbox,

/usr/lib/pymodules/python2.7/matplotlib/transforms.pyc in union(bboxes)
    712         Return a :class:`Bbox` that contains all of the given bboxes.
    713         """
--> 714         assert(len(bboxes))
    715 
    716         if len(bboxes) == 1:

AssertionError: 

```

This goes away if I include any of the standard signals like load_avg.